tests/env/components/ports.go

```diff
@@ -222,7 +224,8 @@ func NewPorts(testId uint16) *Ports {
 		DiscoveryPort:             base + 3,
 		AdminPort:                 base + 4,
 		FakeStackdriverPort:       base + 5,
-		JwtRangeBase:              base + 6,
+		DnsResolverPort:           base + 6,
```

Comment:
Also need to change `portNum` on line 140

Reply:
Thanks!